src/components/modals/ant-management/UpgradeAntModal/UpgradeAntModal.tsx

```diff
@@ -52,15 +52,15 @@ function UpgradeAntModal({
 
   async function upgradeAnts() {
     try {
-      if (!wallet?.arconnectSigner || !walletAddress) {
+      if (!wallet?.contractSigner || !walletAddress) {
         throw new Error('No ArConnect Signer found');
       }
       if (!data?.luaCodeTx) {
         throw new Error('No Lua Code Transaction found');
       }
       setUpgrading(true);
 
-      const signer = createAoSigner(wallet?.arconnectSigner as ContractSigner);
+      const signer = createAoSigner(wallet?.contractSigner as ContractSigner);
```

Comment:
Wondering if we should have wallet.createAoSigner or wallet.createDataItemSigner - can implement the specific chain creation on each respective connector, then we can just do signer = wallet.createDataItemSigner() to get it.